src/interfaces/coral_web/src/utils/getSlugRoutes.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ *
+ * @description This funciton is used to parse the slug from the URL and return the agentId and conversationId.
```

Comment:
```suggestion
 * @description This function is used to parse the slug from the URL and return the agentId and conversationId.
```

also should we move this to above `getSlugRoutes`?

Reply:
yes!